src/App.js

```diff
@@ -90,7 +90,7 @@ function App() {
           dispatch({ type: 'setUser', payload: fullUserData });
         });
       } else {
-        dispatch({ type: 'setUser', payload: null });
+        dispatch({ type: 'setUser', payload: 'visitor' });
```

Comment:
I change that to `visitor` so we'll be able to properly udentify this case in oppose to the case when we we still don't know what is the state of that user (`undefined`)